Ginger/GingerCoreCommon/ReportsLib/ExecutionLoggerConfiguration.cs

```diff
@@ -92,6 +104,56 @@ public long ExecutionLoggerConfigurationMaximalFolderSize
             }
         }
 
+        private ePublishToCentralDB mPublishLogToCentralDB = ePublishToCentralDB.No;
+
+        [IsSerializedForLocalRepository]
+        public ePublishToCentralDB PublishLogToCentralDB 
+        { 
+            get
+            {
+                return mPublishLogToCentralDB;
+            }
+            set
+            {
+                mPublishLogToCentralDB = value;
+                //OnPropertyChanged(nameof(PublishLogToCentralDB));
+            }
+        }
+
+        private eDeleteLocalDataOnPublish mDeleteLocalDataOnPublish = eDeleteLocalDataOnPublish.No;
+
+        [IsSerializedForLocalRepository]
+        public eDeleteLocalDataOnPublish DeleteLocalDataOnPublish
+        {
+            get
+            {
+                return mDeleteLocalDataOnPublish;
+            }
+            set
+            {
+                mDeleteLocalDataOnPublish = value;
+                //OnPropertyChanged(nameof(DeleteLocalDataOnPublish));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)